### PR TITLE
Switch to GHA CI workflow

### LINKED ISSUE #12: Switch to GHA CI workflow
Switch from Travis-CI to GHA CI workflow, as documented at https://gist.github.com/weierophinney/b003e50c3c2667d08076caf31ebd36a4

### PR DESCRIPTION
**Pull request by internalsystemerror**
Switch from Travis CI to use GitHub Actions according to: https://gist.github.com/weierophinney/b003e50c3c2667d08076caf31ebd36a4.

This closes #12

### CODE REVIEW

**Comment by internalsystemerror** [comment]
` spatie/array-to-xml 2.16.0 requires php ^7.4|^8.0 -> your php version (7.3.27) does not satisfy that requirement.`

I can either change PHP support to ^7.4 thus making this error redundant, or I could ignore the job, since the lockfile wouldn't be used when installed by an end user?

File: composer.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
         "psr/container": "^1.0",
         "psr/http-message": "^1.0",
         "psr/http-server-middleware": "^1.0",
-        "spatie/array-to-xml": "^2.3",
+        "spatie/array-to-xml": "^2.3 || ^3.0",
```
Comment (Xerkus) [inline composer.json]: Do tests run without this version bump? I would prefer this done as separate PR if that is a case

Comment (internalsystemerror) [reply composer.json]: They did before this change, happy to revert. They didn't in my local environment however after switching to PHP8, didn't look much further into it though so I'll revert here and submit another PR that is based on this to be merged after.
